### DISCUSSION TITLE
old pod can be automatically when deployment replica num is updated

### DISCUSSION THREAD

**Discussion by AryaCherryLiu** [argoproj/argo-cd#7150] [Q&A]
Hi All,

I Encounter one issue that is:
   after update the deployment replica num in my gitops repo,  when argocd finished the sync action, the old pod still available, and can't be deleted manually

I want to how to resolve.

Thanks

**Comment by jopit** [comment]
Are you incrementing or decrementing the replica count in the deployment?  When I tried this with a simple test, incrementing the count from one to two resulted in one additional pod being created.  Then when I decremented the count again, the new pod was deleted.
